.env.template

```diff
@@ -0,0 +1,12 @@
+KEYCLOAK_ISSUER=https://auth.cuahsi.io/realms/HydroShare
```

Comment:
This has old keys in it.  I also dropped the mongo container because of the confusion and complexity it added (atlas must be used for the triggers and discovery to work).

Reply:
It seems we don't have the `.env.template` file anymore in develop branch. Should I be deleting it from my branch?